plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSource.java

```diff
@@ -160,14 +160,15 @@ public final Block load()
             checkState(batchId == expectedBatchId);
 
             Block block;
+            String parquetFile = parquetReader.getDataSource().getId().toString();
```

Comment:
The id is not necessarily a file, so this is misleading.

Reply:
> The id is not necessarily a file, so this is misleading.

It looks like it is in current codebase.
`HdfsParquetDataSource` is the only implementation of `ParquetDataSource`. And it is constructed with path used as id.

Still changed the wording to allow other identifies.

Reply:
It is currently, but the abstraction doesn't imply a file name. It might change to something else in the future (for instance, it might be a reference to a row group within a file, etc)